fuel-vm/Cargo.toml

```diff
@@ -19,7 +19,7 @@ derivative = "2.2"
 dyn-clone = { version = "1.0", optional = true }
 ethnum = "1.3"
 fuel-asm = { workspace = true }
-fuel-crypto = { workspace = true }
+fuel-crypto = { workspace = true, default-features = false }
```

Comment:
Would this make the VM use the no-std backend by default in all cases? Maybe we should also enable the std feature on fuel-crypto if it's enabled on fuel-vm?

Reply:
Using the no-std version by default could also impact our gas pricing if the performance between the native secp256k1 lib is very different from the pure-rust impl.

Reply:
I'll check that the features are correct, as that seems like a potential issue. It's somewhat hard to verify that the correct crypto backend is getting used, as they have the same external API.

> Using the no-std version by default could also impact our gas pricing if the performance between the native secp256k1 lib is very different from the pure-rust impl.

It certainly does, the `no_std` version is about 5x slower.  The gas pricing shouldn't care about `no_std` performance, as the standard option would be to run the nodes with native secp256k1.

Reply:
Added feature propagation in dc6962a9]